plugin-build/src/test/kotlin/io/sentry/android/gradle/instrumentation/CommonClassVisitorTest.kt

```diff
@@ -0,0 +1,127 @@
+package io.sentry.android.gradle.instrumentation
+
+import io.sentry.android.gradle.instrumentation.fakes.TestSpanAddingParameters
+import io.sentry.android.gradle.instrumentation.util.CatchingMethodVisitor
+import java.io.File
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+import org.objectweb.asm.ClassVisitor
+import org.objectweb.asm.MethodVisitor
+import org.objectweb.asm.Opcodes
+
+class CommonClassVisitorTest {
+
+    class Fixture(private val tmpDir: File) {
+
+        var debug = false
+
+        val sut
+            get() = CommonClassVisitor(
+                Opcodes.ASM9,
+                ParentClassVisitor(),
+                "SomeClass",
+                listOf(TestInstrumentable()),
+                TestSpanAddingParameters(debugOutput = debug, inMemoryDir = tmpDir)
+            )
```

Comment:
this could be a method that takes `debug` as default param=false, so you avoid:
`fixture.debug = true` every time, also, your fixture would be stateless which is what we want.

ideally, there's a single instance of `fixture` instead of a new instance of fixture on `setUp`, this is only necessary if we have states with statics or something.